src/plugins/StatusBar/index.js

```diff
@@ -46,7 +46,8 @@ module.exports = class StatusBar extends Plugin {
       target: 'body',
       hideUploadButton: false,
       showProgressDetails: false,
-      locale: defaultLocale
+      locale: defaultLocale,
+      hideAfterFinish: false
```

Comment:
Would it make sense to default this to `true`?

Reply:
We should also update the docs with the new option.

Reply:
Well, I think setting it to true by default could cause confusion when user doesn't have any other indication that upload has finished successfully

Reply:
i think there's something to say for either--i'm fine with keeping the default `false`, that way people who expect the old behaviour won't be surprised when they upgrade. if we get questions about it we can always change the default later.